websites/A/Anime Genzai/presence.ts

```diff
@@ -0,0 +1,226 @@
+			break;
+
+		case "collection":
+			presenceData.details = `${strings.viewing} collecciones`;
+			presenceData.state = textContent(".collection-detail h1");
+			presenceData.smallImageKey = Assets.Viewing;
+			presenceData.smallImageText = strings.viewing;
+			break;
+
+		case "profile":
+			presenceData.details = `${strings.viewProfile} ${textContent(
+				".profile-content .username"
+			)}`;
+			presenceData.state = textContent(".nav.pt-0 li .active");
+			presenceData.largeImageKey = getImage(".profile-avatar .avatar");
+			presenceData.smallImageKey = Assets.Viewing;
+			presenceData.smallImageText = strings.viewing;
+			break;
+
+		case "search":
+			presenceData.details = privacy
+				? strings.searchSomething
+				: `${strings.searchFor} ${textContent(".subtext").split('"')[1]}`;
+			presenceData.state = textContent(".subtext").split('"')[0].slice(0, -3);
+			presenceData.smallImageKey = Assets.Searching;
+			presenceData.smallImageText = strings.search;
+			break;
+
+		case "serie":
+			presenceData.details = `${strings.viewPage} ${strings.anime}`;
+			presenceData.state =
+				textContent(".pl-md-4 h1") || textContent(".caption-content a h1");
+			presenceData.largeImageKey = getImage(".media");
+			presenceData.smallImageKey = Assets.Viewing;
+			presenceData.smallImageText = strings.viewing;
+			presenceData.buttons = [
+				{
+					label: strings.buttonViewPage,
+					url: href,
+				},
+			];
+
+			if (!privacy && video.currentTime > 0) {
+				presenceData.details = textContent(".caption-content a h1");
+				presenceData.state = `${textContent(
+					".episodes .active .active .episode"
+				)} ${textContent(".episodes .active .active .name")}`;
+			} else if (privacy && video.currentTime > 0)
+				presenceData.details = `${strings.viewing} ${strings.anime}`;
+
+			break;
+
+		case "movie":
+			presenceData.details = `${strings.viewPage} ${strings.anime}`;
+			presenceData.state = textContent(".caption-content h1");
+			presenceData.largeImageKey = getImage(".media");
+			presenceData.smallImageKey = Assets.Viewing;
+			presenceData.smallImageText = strings.viewing;
+			presenceData.buttons = [
+				{
+					label: strings.buttonViewPage,
+					url: href,
+				},
+			];
+
+			if (video.currentTime > 0) presenceData.details = strings.viewMovie;
+			break;
+	}
+
+	if (!privacy && video.currentTime > 0 && path !== "tv-channel") {
+		presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+		presenceData.smallImageText = video.paused ? strings.pause : strings.play;
+		presenceData.buttons = [
+			{
+				label: strings.watchVideoButton,
+				url: href,
+			},
+		];
+
+		if (video.paused || !time) {
+			delete presenceData.startTimestamp;
+			delete presenceData.endTimestamp;
+		} else {
+			[presenceData.startTimestamp, presenceData.endTimestamp] =
+				presence.getTimestamps(video.currentTime, video.duration);
+		}
+	}
+
+	if (!logo || privacy) presenceData.largeImageKey = Assets.Logo;
```

Comment:
You could add a check to see if it has even changed (Optional tho):
```suggestion
	if ((privacy || !logo) && presenceData.largeImageKey !== Assets.Logo) presenceData.largeImageKey = Assets.Logo;
```